presto-hive-hadoop2/bin/common.sh

```diff
@@ -28,7 +28,7 @@ function hadoop_master_ip() {
 }
 
 function check_hadoop() {
-  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container | grep .)
```

Comment:
i'd add `| grep .` inside `hadoop_master_container` function. Any reason not to?